framework/resources/Functional/union_all/prq_union_all/prqUAllprq_36.sql

```diff
@@ -0,0 +1,9 @@
+drop table if exists dfs.union_all.`MyTbl_1`;
+drop table if exists dfs.union_all.`MyTbl_2`;
+drop view if exists dfs.union_all.`MyView1`;
+create table dfs.union_all.MyTbl_1 as (select 1 as Col);
+create table dfs.union_all.MyTbl_2 as (select 2 as Col);
+use dfs.tmp;
```

Comment:
I'm a bit confused. We're using dfs.tmp here and dfs.union_all above and below

Reply:
Fixed.